x-pack/plugins/infra/public/components/saved_views/toolbar_control.tsx

```diff
@@ -95,10 +112,7 @@ export function SavedViewsToolbarControls<ViewState>(props: Props<ViewState>) {
           />
         </EuiButtonEmpty>
         <EuiButtonEmpty iconType="importAction" onClick={loadViews} data-test-subj="loadViews">
```

Comment:
I feel like the `importAction` icon doesn't make sense anymore now that this button doesn't say "Load." Maybe something that communicates "This is the current view" better?

`grid` seems like the closest thing in EUI to the look of the waffle map. `node` might also work even though it's not a square.